include/utility/includes.h

```diff
@@ -0,0 +1,10 @@
+  
+#ifndef __RF24_INCLUDES_H__
+#define __RF24_INCLUDES_H__
+
```

Comment:
IMHO we should leave the RF24 library and put in the makefile only a step to copy all the headers file in a new directory called `include` which will be then removed at the clean target .